pkg/config/config.go

```diff
@@ -56,3 +60,36 @@ func GetExecutorEnvVarConfOptions(app *v1beta2.SparkApplication) []string {
 func GetPrometheusConfigMapName(app *v1beta2.SparkApplication) string {
 	return fmt.Sprintf("%s-%s", app.Name, PrometheusConfigMapNameSuffix)
 }
+
+// GetK8sConfigMap gets the cmName configmap in app.Namespace and returns the configMapPath
+func GetK8sConfigMap(app *v1beta2.SparkApplication, cmName string) (string, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		glog.Errorf("%v", err)
+		return "", err
+	}
+
+	clientset, err := res.NewForConfig(config)
+	if err != nil {
+		glog.Errorf("%v", err)
+		return "", err
+	}
+
+	configMapsInNamespace := clientset.ConfigMaps(app.Namespace)
+	userProvidedConfigMap, err := configMapsInNamespace.Get(cmName, metav1.GetOptions{})
+
```

Comment:
Note that the client-go package version is 0.19.6 in the master branch, so it should add context.TODO() to the first argument.